roles/confluent.variables/defaults/main.yml

```diff
@@ -302,9 +311,10 @@ kafka_broker_rest_proxy_enabled: "{{confluent_server_enabled}}"
 
 #### Schema Registry Variables ####
 
-### Only use to customize Linux User Schema Registry Service runs with. User must exist on host.
+### Only use to customize Linux User Schema Registry Service runs with. Default user is cp-schema-registry
 schema_registry_user: "{{schema_registry_default_user}}"
-### Only use to customize Linux Group Schema Registry Service user belongs to. Group must exist on host.
+
+### Only use to customize Linux Group Schema Registry Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Schema Registry Service user belongs to. Default group is confluent.